RDKShell/RDKShell.cpp

```diff
@@ -995,6 +993,13 @@ namespace WPEFramework {
             {
                 gWillDestroyEventWaitTime = atoi(willDestroyWaitTimeValue); 
             }
+
+            if (Core::ERROR_NONE != subscribeForSystemEvent("onSystemPowerStateChanged"))
```

Comment:
avoid doing subscription call here as, if it goes to activation, it may delay rdkshell initialization. Isntead during initialization, start the timer with minimal delay

Reply:
Added 2 seconds delay.